BUILD

```diff
@@ -0,0 +1,8 @@
+py_test(
```

Comment:
I'm not sure what the best practice here is, but do you think this BUILD file should be in the test subdir?

Reply:
I think it makes sense conventionally to be at the root, similar to a Makefile. You'd typically run your make/bazel commands from the root of the repo.

Also, `BUILD` references files in `scripts` and `identifiers`, too, so to access those, they need to be underneath a directory with a `BUILD` file. So, if there's no top-level `BUILD`, we'd need a separate `BUILD` in each subdir. It seems cleaner this way.

Reply:
Good point about the different directory access and having it be used in a more similar style to a Makefile.